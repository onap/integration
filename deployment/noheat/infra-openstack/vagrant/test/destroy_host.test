#!/bin/sh

export HOST_NAME='operator0'

export VAGRANT_CWD='..'

set_up() {
    vagrant up --provision-with=run_playbook_create
    vagrant up --provision-with=run_playbook_destroy
}

check() {
    local host="$1"
    vagrant ssh operator --no-tty -c \
        "export OS_CLOUD=openstack; openstack server list -fcsv" \
        | grep "$host" \
        || echo "Host ${host} not found."
}

set_up >/dev/null # drop provisioning output
check "$HOST_NAME"
