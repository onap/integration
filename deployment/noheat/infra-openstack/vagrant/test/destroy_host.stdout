Host operator0 not found.
